spring-cloud-task-samples/task-events/README.adoc

```diff
@@ -0,0 +1,33 @@
+= Task Events
+
+This is a task application that emits events on a channel named `task-events`
+
+== Requirements:
+
+* Java 7 or Above
+
+== Build:
+
+[source,shell,indent=2]
+----
+$ ./mvnw clean install
```

Comment:
Works for me on a clean repo.  Based on this, I'm wondering if you have an odd git configuration: http://stackoverflow.com/questions/2920416/configure-bin-shm-bad-interpreter